SongLab/Components/Home/HomeViewModel.swift

```diff
@@ -13,6 +13,7 @@ import SwiftUI
 class HomeViewModel: ObservableObject {
     
     //MARK: - API
+    @AppStorage("metronomeActive") var metronomeActive: Bool = false
```

Comment:
Same question as above. It looks like you are using @AppStorage to create a global variable.